spack-environments/isambard-cascadelake/compute-node/spack.yaml

```diff
@@ -42,30 +42,28 @@ spack:
   - compiler:
       spec: gcc@8.1.0
       paths:
-        cc: cc
-        cxx: CC
-        f77: ftn
-        fc: ftn
+        cc: /opt/gcc/8.1.0/bin/gcc
+        cxx: /opt/gcc/8.1.0/bin/g++
+        f77: /opt/gcc/8.1.0/bin/gfortran
+        fc: /opt/gcc/8.1.0/bin/gfortran
       flags: {}
       operating_system: rhel8
       target: any
       modules:
-      - PrgEnv-gnu
```

Comment:
I can't find this module anymore, I wonder if that has been removed at some point.